assets/src/components/GoogleAnalyticsTracking.js

```diff
@@ -1,24 +1,25 @@
 import { useState, useEffect } from 'react'
-import GoogleAnalytics from 'react-ga'
+import GoogleAnalytics from 'react-ga4'
 
 function GoogleAnalyticsTracking (props) {
-  const {
-    gaId
-  } = props
+  const { gaId, cspNonce } = props
 
   const [initialized, setInitialized] = useState(false)
   const [previousPage, setPreviousPage] = useState(null)
 
   if (gaId && !initialized) {
     setInitialized(true)
-    GoogleAnalytics.initialize(gaId)
+    GoogleAnalytics.initialize([{
+      trackingId: gaId,
+      gaOptions: { nonce: cspNonce, cookieFlags: 'SameSite=None; Secure' }
```

Comment:
The `nonce` is not strictly necessary because we are using `unsafe-inline` but using this now will set us up for the future if we are able to get the rest of the application to use this approach.